packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -264,8 +264,8 @@ export function TableNG(props: TableNGProps) {
     });
 
     if (footerOptions?.show && footerOptions.reducer.length > 0) {
-      if (footerOptions.countRows) {
-        footerItems = ['Count', rows.length.toString()];
+      if (footerOptions.countRows && footerOptions.reducer[0] === ReducerID.count) {
```

Comment:
Good catch with this one! I did see this behavior before, where the user sets countRows to true, then switches the reducer and calculations aren't done for all fields, but forgot to capture the issue. Thanks for catching it here!